dev-manual/api/api-reference.rst

```diff
@@ -9,20 +9,138 @@ Quickly review all available resources for the Archivematica API and Storage Ser
 Archivematica API
 ==================
 
-The Archivematica API provides access to the following categories of resources:
+The Archivematica API provides:
 
-    * Transfer
-    * Ingest
-    * Administration
-    * Unit
-    * Other
-    * Beta endpointsq
+    * Coverage of some basic workflow functionality.
+    * Proxy support to the Storage Service.
```

Comment:
I wasn't sure what proxy support was at first, but see it's about the dashboard providing some functions that one might normally anticipate to be provided by the storage service. I hadn't thought about it like a proxy before :)